backend/main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+)
+
+func uploadHandler(w http.ResponseWriter, r *http.Request) {
+	// Log the incoming request for debugging purpose
+	fmt.Println("Upload request received")
+
+	// Parse the form data
+	err := r.ParseMultipartForm(10 << 20) // Limit upload to 10MB
+	if err != nil {
+		http.Error(w, "Unable to parse form", http.StatusBadRequest)
+		fmt.Println("Error parsing form:", err)
+		return
+	}
+
+	// Retrieve the file from the form
+	file, _, err := r.FormFile("video")
+	if err != nil {
+		http.Error(w, "Error retrieving file", http.StatusBadRequest)
+		fmt.Println("Error retrieving file:", err)
+		return
+	}
+	defer file.Close()
+
+	// Create a file to store the uploaded video
+	outFile, err := os.Create(filepath.Join("uploads", "recording.webm"))
```

Comment:
Problems in this
- use a constant for now for 'uploads' string
   - later it will be added to configuration
- everyfile will bee recording.web, hence every upload will overwrite previous file !!
 - generate a uuid and use that as a filename
- .webm extension - you dont know what extension video is the user uploading and can change in future, dont assume its webm, use the extension of the file uploaded by client (webapp)